scripts/system/libraries/controllerDispatcherUtils.js

```diff
@@ -600,6 +602,10 @@ worldPositionToRegistrationFrameMatrix = function(wptrProps, pos) {
     return offsetMat;
 };
 
+handsAreTracked = function () {
+    return Controller.getPoseValue(Controller.Standard.LeftHandIndex3).valid ||
```

Comment:
why do these check Index3?  is it arbitrary?

Reply:
I expect that a finger joint is used (rather than palm of hand, say) to ensure that fingers are being tracked so that hand gestures (e.g., touching tips of fingers together) can be detected and acted upon. (Even though this seems not to be working at present.)